CHANGELOG.md

```diff
@@ -4,7 +4,20 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v8.0.2](https://github.com/voxpupuli/puppet-php/tree/v8.0.2) (2021-09-05)
+## [v8.0.3](https://github.com/voxpupuli/puppet-php/tree/v8.0.3) (2022-03-20)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-php/compare/v8.0.2...v8.0.3)
+
+**Fixed bugs:**
+
+- Allow a setting to be undef to force absent. [\#647](https://github.com/voxpupuli/puppet-php/pull/647) ([jadestorm](https://github.com/jadestorm))
```

Comment:
This *sounds* like en enhancement, but is indeed a bugfix…  Maybe it's worth to rephrase the RP title to emphasis on the fix e.g. "Fix forcing a php::setting to be absent by setting its value to undef"?

But I can really live without it :-)

Reply:
Sounds good!